modules/registration/registration_plugin.rb

```diff
@@ -0,0 +1,8 @@
+module Proxy::Registration
+  class Plugin < ::Proxy::Plugin
+    http_rackup_path  File.expand_path("http_config.ru", File.expand_path(__dir__))
+    https_rackup_path File.expand_path("http_config.ru", File.expand_path(__dir__))
```

Comment:
Since 88fbc8e67d665e2c3b19acb53b31ff30acf078b7 there's a new preferred way
```suggestion
    rackup_path File.expand_path('http_config.ru', __dir__)
```